generators/ruby/faraday/templates/README.mustache

```diff
@@ -54,52 +59,46 @@ All 3 error classes provide the `response_code`, `response_body`, `json_body`, `
 
 ```ruby
 def create_applicant
-  onfido.applicant.create(params)
+  {{{gemName}}}.applicant.create(params)
```

Comment:
This is the old library. We should check which errors we're getting back with the new library to provide a correct example.